netbox/dcim/forms/bulk_edit.py

```diff
@@ -574,7 +575,31 @@ class DeviceTypeBulkEditForm(NetBoxModelBulkEditForm):
     nullable_fields = ('part_number', 'airflow', 'weight', 'weight_unit', 'description', 'comments')
 
 
+class ModuleTypeProfileBulkEditForm(NetBoxModelBulkEditForm):
+    schema = forms.JSONField(
+        label=_('Schema'),
+        required=False
+    )
```

Comment:
I noticed that `schema` values of `null` and `""` don't affect any change on the items selected. However, I noticed that `{}` does essentially clear the schemas of the selected items.

This might be the intended behavior, but was not super intuitive.

Reply:
Relatedly, `true` and `false` literals were accepted as valid input for this field, even though they don't necessarily make sense.

Perhaps values should be locked down to some representation of no schema (`null` or `{}` perhaps) and some schema that is at least of type object. Not sure, though, just thinking out loud.